commands/inventory/add_agent_postgres_exporter.go

```diff
@@ -54,17 +56,42 @@ type addAgentPostgresExporterCommand struct {
 	Password            string
 	CustomLabels        string
 	SkipConnectionCheck bool
-	TLS                 bool
-	TLSSkipVerify       bool
 	PushMetrics         bool
 	DisableCollectors   string
+
+	TLS           bool
+	TLSSkipVerify bool
+	TLSCAFile     string
+	TLSCertFile   string
+	TLSKeyFile    string
 }
 
 func (cmd *addAgentPostgresExporterCommand) Run() (commands.Result, error) {
 	customLabels, err := commands.ParseCustomLabels(cmd.CustomLabels)
 	if err != nil {
 		return nil, err
 	}
+
+	var tlsCa, tlsCert, tlsKey string
+	if cmd.TLSCAFile == "" || cmd.TLSCertFile == "" || cmd.TLSKeyFile == "" {
```

Comment:
this is a very weird behaviour. TLS might be enabled without custom certificates. And here we even don't check for TLS value, so it always returns error even if tls is off.

Reply:
Yes, got it.
Alined with MySQL implementation https://github.com/percona/pmm-admin/blob/master/commands/management/add_mysql.go#L146:L146